dist/index.mustache

```diff
@@ -0,0 +1,124 @@
+                        Reserved
+                        Keywords</label>
+                </div>
+                <div>
+                    <div id="doc"></div>
+                </div>
+                <script charset="utf-8">
+                    require(["../../lib/docson", "../../lib/main", "../../lib/jquery"], function (docson, main) {
+                        $(function () {
+                            var schema;
+                            $.get('../collection.json', function (response) {
+                                schema = response;
+                                docson.templateBaseUrl = "../../templates";
+                                docson.doc("doc", schema);
+                            });
+
+                            $('#show-internal-check').change(function () {
+                                var unchecked = !this.checked;
+                                $.get('../collection.json', function (schema) {
+                                    docson.templateBaseUrl = "../../templates";
+                                    if (unchecked) {
+                                        main.removePrivate(schema);
+                                    }
+                                    console.log(JSON.stringify(schema, null, 2));
+                                    docson.doc("doc", schema);
+                                });
+                            });
+                        });
+                    });
+
+                </script>
+            </div>
+            <div>
+                <h3>Legend</h3>
+                <table class="table table-striped">
+                    <tbody>
+                        <tr>
+                            <td><span class="required">propertyName</span></td>
+                            <td>A Required property.</td>
+                        </tr>
+                        <tr>
+                            <td><span class="signature-type-string"><strong>dataType</strong></span></td>
+                            <td>Denotes the data type of a property, e.g "string" or "number".</td>
+                        </tr>
+                        <tr>
+                            <td><span class="type-enum"><strong>variableValue</strong></span></td>
+                            <td>One of a few valid values.</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div>
+                <p>
+                    The raw schema is located at
+                    <a href="http://schema.getpostman.com/json/collection/{{ version }}/collection.json" target=“_blank”>
+                        http://schema.getpostman.com/json/collection/{{ version }}/collection.json
+                    </a>
+                </p>
+            </div>
+        </div>
+    </div>
+
+    <div class="footer-container open-sans">
+        <span>&copy; 2018 Postdot Technologies, Inc.</span>
```

Comment:
We don't want to update the 2018 every year. Is there no way to get the current year in here?